editorSupport/sublime-reason/Reason.py

```diff
@@ -28,7 +28,6 @@ def reason_command_line(file_name):
 
     return [
         refmt,
-        '--use-stdin', 'true',
         '--parse', 're',
         '--interface', is_interface(file_name),
```

Comment:
Nevermind, it's reformatting as a binary which is what the method claims to be doing and so this works.